rust_src/crates/js/Cargo.toml

```diff
@@ -14,13 +14,18 @@ lsp_json = { version = "0.1.0", path = "../lsp_json" }
 libc = "0.2.95"
 lazy_static = "1.2"
 futures = "0.3"
+deno = { git = "https://github.com/DavidDeSimone/deno", branch = "emacs-ng@1.30.3" }
```

Comment:
A finalized version of the PR will use emacs-ng's deno fork vice my own

Reply:
> Updating git repository `https://github.com/DavidDeSimone/deno`

This line takes forever like before. Maybe we should find a better way?

Reply:
Let me hack around on this and see what I can do.

Reply:
@declantsien I made some changes for the deno submodule to reduce clone size - can you let me know if it is still taking a long time for initial clone for you?